README.md

```diff
@@ -1,32 +1,59 @@
-API Endpoints
+# Covents Backend 
+
+  - BE team:  Dan Reardon and Kelly Bard
```

Comment:
I might add links to your GH profiles here!

Reply:
Agreed

Reply:
Will do on next revision